contents/shells/bash.md

```diff
@@ -0,0 +1,179 @@
+<frontmatter>
+  title: Introduction to bash shell
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
```

Comment:
Overall, there seems to be a mix up between features and benefits. Consider this structure:

What is the shell?
* It is a CLI for the OS (is that correct?). Here, we focus on Bash shell.
* Bash supports a Turing-complete language for writing scripts.
* Bash supports combining of scripts via stream redirection.

Why use shell?
* Benefit: it is faster
* Benefit: can automate tasks
* Benefit: it is more powerful than a GUI in the hands of an expert

How to get started?
* Learn as you go
* Some beginner resources: ...
* Use man when you need more info